server/middlewares.js

```diff
@@ -1,18 +1,38 @@
 const logger = require('morgan')
+const CONFIG = require('config')
+const passport = require('./middlewares/passport')
 
 const cors = function (req, res, next) {
-  res.header('Access-Control-Allow-Origin', '*')
+  res.header('Access-Control-Allow-Credentials', true)
+  res.header('Access-Control-Allow-Origin', req.headers.origin)
   res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE')
-  res.header('Access-Control-Allow-Headers', 'Content-Type')
-  next()
+  res.header('Access-Control-Allow-Headers', 'X-Requested-With, X-HTTP-Method-Override, Content-Type, Accept')
+  if ('OPTIONS' == req.method) {
+    res.send(200)
+  } else {
+    next()
+  }
 }
 
 module.exports = {
   common: [
     require('body-parser').json(),
     require('method-override')(),
     require('errorhandler')({ dumpExceptions: true, showStack: true }),
-    cors
+    cors,
+    passport.initialize,
+    passport.session({
+      cookie: {
+        key: CONFIG.get('store.host'),
```

Comment:
This is the hostname of the CouchDB instance. Why is it added or serialised into the Passport cookie?